CMakeLists.txt

```diff
@@ -12,6 +12,9 @@ endif (POLICY CMP0135)
 
 project(mender)
 
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
```

Comment:
AFAICT, I think we don't to do this globally.

Reply:
what do you mean? That we shouldnt do it globally.
that this does not change it globally?

Reply:
ooops, dropped some words. I was trying to say that we don't **want** to do this globally :sweat_smile: